install/mysql/glpi-empty.sql

```diff
@@ -1380,6 +1380,26 @@ INSERT INTO `glpi_configs` VALUES ('206','core','purge_all','0');
 INSERT INTO `glpi_configs` VALUES ('207','core','purge_user_auth_changes','0');
 INSERT INTO `glpi_configs` VALUES ('208','core','purge_plugins','0');
 INSERT INTO `glpi_configs` VALUES ('209','core','display_login_source','1');
+INSERT INTO `glpi_configs` VALUES ('210','core','impact_assets_list','');
+
+### Dump table glpi_impacts
+
+CREATE TABLE `glpi_impacts` (
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`source_asset_type` VARCHAR(255) NOT NULL DEFAULT '' COLLATE 'utf8_unicode_ci',
```

Comment:
Some automatic relation handling is made based on field names. So it is important to fits existing naming rules.
For example: https://github.com/glpi-project/glpi/blob/9.5/bugfixes/inc/commondbrelation.class.php#L109
```suggestion
	`itemtype_source` VARCHAR(255) NOT NULL DEFAULT '' COLLATE 'utf8_unicode_ci',
```